_security/configuration/demo-configuration.md

```diff
@@ -0,0 +1,115 @@
+---
+layout: default
+title: Setting up a demo configuration
+parent: Configuration
+nav_order: 4
+---
+
+# Setting up a demo configuration
+
+Welcome to the OpenSearch Security plugin's demo configuration setup guide. This tool provides a quick and easy way to replicate a production environment for testing purposes. The demo configuration includes the setup of security-related components such as internal users, roles, role mappings, audit configuration, basic authentication, tenants, and allow lists.
+
+
+The demo configuration tool performs the following tasks:
+
+1. Configures security settings which are then loaded into the security index.
+2. Generates demo certificates.
+3. Updates the `opensearch.yml` file to add security-related settings.
+
+## Installing demo configuration
+
+The demo configuration is automatically called as part of the setup for each supported distribution of OpenSearch. Following are the instructions per distribution.
+
+**Note**: Starting from OpenSearch 2.12, a custom admin password is required to install demo configuration. If none is provided, the cluster would fail to start. Note, that this change affects fresh clusters only. Already setup clusters are not affected, as they already have `opensearch.yml` configured, and so the installation tool will not run. 
```

Comment:
```suggestion
**Note**: Starting with OpenSearch 2.12, a custom admin password is required in order to install the demo configuration. If none is provided, the cluster will fail to start. Note that this change only affects new clusters. Existing clusters are not affected because they already have `opensearch.yml` configured, so the installation tool will not run. 
```